include/extractor/profile_properties.hpp

```diff
@@ -12,7 +12,7 @@ struct ProfileProperties
 {
     ProfileProperties()
         : traffic_signal_penalty(0), u_turn_penalty(0), continue_straight_at_waypoint(true),
-          use_turn_restrictions(false), left_hand_driving(false)
+          use_turn_restrictions(false), left_hand_driving(false), max_speed(0)
```

Comment:
That was just a question, since I was unsure how this is exposed. I remembered problems coming up with a change @danpat did some while back. If we don't expose Profile Properties anywhere but in lua, thats totally fine.